_posts/2019-10-21-vscode-quarkus-0.1.0.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: 'Quarkus Tools for Visual Studio Code - 1.1.0 release'
+date: 2019-10-21
+synopsis: New features for Quarkus Tools for Visual Studio Code.
+author: dakwon
+---
+:blank: pass:[ +] 
+:imagesdir: /assets/images/posts/quarkus-vs-code-1.1.0
+
+{blank}
+
+== What’s new with Quarkus Tools for Visual Studio Code
+The 1.1.0 release of https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Quarkus Tools for Visual Studio Code]
+offers more language features for the application.properties file and new additions to existing
+features like validation and snippets support.
+
+For a list of all changes, please refer to the https://github.com/redhat-developer/vscode-quarkus/blob/master/CHANGELOG.md[changelog].
+
+{blank}
+
+== New features
+* link:#value-type-validation-for-application-properties[Value type validation for application.properties]
+* link:#go-to-definition-support-for-config-properties-in-application-properties[Go to definition support for application.properties]
+* link:#formatting-support-for-application-properties[Formatting support for application.properties]
+* link:#retrieve-test-properties-for-an-application-properties-file-in-srctestresources[Retrieve test properties for an application.properties file in `src/test/resources`]
+* link:#new-snippets[New snippets (Quarkus datasource & Jaeger tracer)]
+
+{blank}
+
+=== Value type validation for application.properties
+For property values, there will now be an error diagnostic
+if the expected type does not match the value type.
+
+image::01_type_validation.gif[alt="Validate property value types"]
+
+For the example above, `quarkus.http.cors` expects a `boolean` and
+`quarkus.http.port` expects an `int`.
+
+The `item.price` config property was defined in a resource class to 
+expect a `float` like so: 
+
+.MyResource.java
+[source,java]
+----
+@ConfigProperty(name = "item.price")
+float price;
+----
+
+{blank}
+
+=== Go to definition support for config properties in application.properties
```

Comment:
I personally still don't understand the use case of going to the extension property internal code via navigation.
So I'd focus on the item.price example which has relevance to the user.